test/DepthImageToLaserScanTest.cpp

```diff
@@ -63,10 +65,10 @@ TEST(ConvertTest, setupLibrary)
   dtl_.set_scan_height(scan_height);
   const std::string output_frame = "camera_depth_frame";
   dtl_.set_output_frame(output_frame);
-  
-  depth_msg_.reset(new sensor_msgs::Image);
-  depth_msg_->header.seq = 42;
-  depth_msg_->header.stamp.fromNSec(1234567890);
+
+  depth_msg_.reset(new sensor_msgs::msg::Image);
+  depth_msg_->header.stamp.sec = 0;
```

Comment:
The problem is that header.stamp doesn't have a fromNSec() method anymore.  I just filled out the structure with 0 to make my life easy, since it isn't used in the rest of the test.